app/models/concerns/smooch_search.rb

```diff
@@ -146,7 +146,7 @@ def search_for_similar_published_fact_checks_no_cache(type, query, team_ids, aft
           text = [link.pender_data['description'].to_s, text.to_s.gsub(/https?:\/\/[^\s]+/, '').strip].max_by(&:length)
         end
         return [] if text.blank?
-        words = text.split(/\s+/)
+        words = Bot::Alegre.get_number_of_words(text)
```

Comment:
FYI @caiosba I think we can and should use the new function here in Smooch Bot as well. Let me know your thoughts, please.

Reply:
I agree @computermacgyver